tests/opkg.py

```diff
@@ -56,3 +56,31 @@ def runTest(self):
             board.sendline("\nopkg install {}".format(pkg))
             board.expect("Configuring {}".format(pkg))
             board.expect(prompt)
+            board.sendline('\nopkg list-installed | grep {}'.format(pkg))
+            try:
+                board.expect('{} - '.format(pkg)) # pass if installed
+            except:
+                assert False # fail if not installed
+
+class OpkgRemove(rootfs_boot.RootFSBootTest):
+    '''Opkg is able to remove selected packages and their dependencies, if any'''
+    def runTest(self):
+        # One package per feed: packages, openwrt-routing, openwrt-managements,
+        # ci40-platform-feed, telephony, luci, kernel-module
+        packages = [ "nano", "mrd6", "libssh", "glog", "miax", "luci-mod-rpc" , "kmod-usb-net-qmi-wwan" ]
```

Comment:
Sounds good then, I guess in the future we can think about adding some kind of a dictionary with packages if we need to remove/install them depending on the feeds in different tests.